MetaMask/metamask-extension
# Enable buying MATIC on Polygon, BNB on BSC, AVAX on Avalanche, FTM on Fantom, CELO on Celo, and respective stablecoins  on Transak

Fixes: [#](https://github.com/MetaMask/metamask-extension/issues/13351)

Explanation:  

We'd like to enable purchasing MATIC and BNB for those networks via just the Transak route.

Manual testing steps:  
  - Switch to BSC and Polygon networks
  - See the "Buy" button enabled
  - Click the Transak link and ensure the proper currency and network are shown.
